src/go-slang/parser/go.pegjs

```diff
@@ -225,7 +232,7 @@ ReceiveExpression
 /* Go Statement */
 
 GoStatement
-    = GO_TOKEN __ call:Expression EOS {
+    = GO_TOKEN __ call:CallExpression EOS {
```

Comment:
this is to temporarily fix a bug where the call expression of a `GoStatement` is parsed as a `BinaryExpression` if we have a `ReceiveExpression`* afterwards (see below)

```go
func main() {
    go task()
    <-ch // this will be parsed as a `BinaryExpression` instead of a `ExpressionStatement`
}
```

*the error happens for all unary operators